### PR TITLE
Remove hyperlinks in BAL file code comments in all BBEs

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Remove hyperlinks in BAL file code comments in the BBEs below.

- [x] xslt-transformation
- [x] all WebSocket BBEs
- [x] all TCP BBEs
- [x] all security BBEs
- [x] all RabbitMQ BBEs
- [x] all NATS BBEs
- [x] all Kafka BBEs
- [x] all HTTP BBEs
- [x] all gRPC BBEs
- [x] all graphQL BBEs

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by Bhashinee — APPROVED** [review]

**Review by dilanSachi — APPROVED** [review]

**Review by TharmiganK — APPROVED** [review]

File: examples/websocket-client-basic-auth/websocket_client_basic_auth.bal [hunk 1]
```diff
@@ -3,8 +3,8 @@ import ballerina/websocket;
 
 // Defines the WebSocket client to call the Basic Auth secured APIs.
 // The client is enriched with the `Authorization: Basic <token>` header by
-// passing the [`websocket:CredentialsConfig`](https://lib.ballerina.io/ballerina/websocket/latest/records/CredentialsConfig) for the `auth` configuration of the
-// client.
+// passing the `websocket:CredentialsConfig` for the `auth` configuration of the client.
+// FOr details, see https://lib.ballerina.io/ballerina/websocket/latest/records/CredentialsConfig.
```
Comment (MohamedSabthar) [inline examples/websocket-client-basic-auth/websocket_client_basic_auth.bal:7]: Typo `FOr`

**Review by MohamedSabthar — COMMENTED** [review]
Found typo in 2 places

File: examples/websocket-client-basic-auth/websocket_client_basic_auth.bal (hunk 1 above)
Comment (MohamedSabthar) [inline examples/websocket-client-basic-auth/websocket_client_basic_auth.bal:7]:
```suggestion
// For details, see https://lib.ballerina.io/ballerina/websocket/latest/records/CredentialsConfig.
```

File: examples/websocket-service-mutual-ssl/websocket_service_mutual_ssl.bal [hunk 2]
```diff
@@ -3,7 +3,8 @@ import ballerina/websocket;
 
 // A WebSocket listener can be configured to accept new connections that are
 // secured via mutual SSL.
-// The [`websocket:ListenerSecureSocket`](https://lib.ballerina.io/ballerina/websocket/latest/records/ListenerSecureSocket) record provides the SSL-related listener configurations.
+// The `websocket:ListenerSecureSocket` record provides the SSL-related listener configurations.
+// FOr details, see https://lib.ballerina.io/ballerina/websocket/latest/records/ListenerSecureSocket.
```
Comment (MohamedSabthar) [inline examples/websocket-service-mutual-ssl/websocket_service_mutual_ssl.bal:7]:
```suggestion
// For details, see https://lib.ballerina.io/ballerina/websocket/latest/records/ListenerSecureSocket.
```

**Review by MohamedSabthar — APPROVED** [review]

**Review by MadhukaHarith92 — APPROVED** [review]

**Review by kalaiyarasiganeshalingam — APPROVED** [review]

File: examples/grpc-service-ssl-tls/grpc_service_ssl_tls.bal [hunk 3]
```diff
@@ -3,8 +3,9 @@ import ballerina/grpc;
 // A gRPC listener can be configured to communicate through SSL/TLS as well.
 // To secure a listener using SSL/TLS, the listener needs to be configured
 // with a certificate file and a private key file for the listener.
-// The [`grpc:ListenerSecureSocket`](https://lib.ballerina.io/ballerina/grpc/latest/records/ListenerSecureSocket) record
+// The `grpc:ListenerSecureSocket` record 
 // provides the SSL-related listener configurations of the listener.
```
Comment (daneshk) [inline examples/grpc-service-ssl-tls/grpc_service_ssl_tls.bal:7]:
```suggestion
// The `grpc:ListenerSecureSocket` record provides the SSL-related listener configurations of the listener.
```

**Review by daneshk — APPROVED** [review]
LGTM

**Pull request merged**